pilz_trajectory_generation/CMakeLists.txt

```diff
@@ -430,7 +452,9 @@ ${catkin_LIBRARIES} ${PROJECT_NAME}_test)
   )
 
   target_link_libraries(unittest_pilz_command_planner
-    ${catkin_LIBRARIES} ${PROJECT_NAME}_test)
+    ${catkin_LIBRARIES}
+    ${PROJECT_NAME}
+  )
```

Comment:
Unittest `pilz_command_planner` cannot be successfully build and executed without building the complete ROS-worksapce with `catkin_make` beforehand. But maybe this was not an objective, then everything is fine with me.
(Please note I did not test every test. I just randomly picked one and tested if I can execute it without building the entire ROS-workspace)